doc/playbook.rst

```diff
@@ -148,6 +149,46 @@ At the HEAD of the ``main`` branch, create and push a tag with the semantic vers
    git tag -s X.Y.Z -m "X.Y.Z"
    git push --tags
 
+Patch Releases and Release Branches
+-----------------------------------
+
+During commissioning and operations, it may be necessary to quickly deploy a bug fix without making any other changes that might potentially introduce new breakages.
+This can be done using a patch version (``X.Y.Z``).
+If there have been unrelated changes committed since the last release, you will need to isolate the bug fixes on a release branch.
+
```

Comment:
Would it be useful to put the link https://developer.lsst.io/work/backports.html  directly in the notebook as a reference how this process is based on?

Reply:
I think that might be more confusing than not -- there are things there that we don't want to deal with (multiple approvals, multiple release candidates, etc.), and our build system works differently. As I mentioned on Jira, our decision flow is (might be?) a bit different too -- at least this time, we decided to backport DM-46520 as part of the process of making a release, and not as part of the process of implementing DM-46520.

(That makes me think I should switch the order of "if you have a branch..." and "cherry-pick", but I can't think of a natural wording.)